### ISSUE TITLE
Rename "Add New SSH Host..."

### ISSUE THREAD

**Issue by PlethoraChutney** [microsoft/vscode#227472]
In the "Remote-SSH: Connect to Host..." dialog, I'd like to suggest/request that the "Add New SSH Host..." option is renamed.

We recently had to change which node of a cluster we log into from `node_a` to `node_b`. To connect to `node_b`, **100%** of VS Code users (including myself 🙃) clicked "Add New SSH Host..." and entered `node_b`. This causes VSC to add the useless lines
```
Host node_b
    HostName node_b
```
to the top of `~/.ssh/config`, which breaks the existing config. Now users can't even SSH into `node_b` from the terminal, which was working before attempting to connect using VSC.

Users were all surprised to learn you could simply type `node_b` into the "Connect to Host..." dialog, and that this was not considered "adding a new host".

I think this confusion would be resolved if the dialog included a line like "Enter hostname" and a separate line named "Add new host to .ssh/config". The latter would have the functionality that's currently under "Add New SSH Host..."

Thank you!

**Comment by roblourens** [comment]
What is `node_b`, that's a hostname?

> which breaks the existing config

How does it break the existing config?

**Comment by roblourens** [comment]
Or was it already an entry in the ssh config?

**Comment by PlethoraChutney** [comment]
Yes, sorry, that's a hostname. We have existing entries in the ssh config which correctly route hostnames matching a pattern through a login node, so adding an entry specifically matching that hostname prevents that routing from working.

**Comment by PlethoraChutney** [comment]
More explicitly, here's what's essentially in .ssh/config:

```
Host login
    HostName login.domain

Host node_*
    ProxyCommand ssh login -W %h:%p
    Hostname %h.domain
```

So adding the HostName lines breaks the `-W` routing through `login.domain`.
